gcd/src/main/java/org/spine3/sample/gae/Application.java

```diff
@@ -55,8 +55,8 @@ public static void main(String[] args) {
         final LocalDatastoreStorageFactory storageFactory = LocalDatastoreStorageFactory.getDefaultInstance();
         storageFactory.getOptions().setEventIteratorPageSize(ITERATOR_PAGE_SIZE);
```

Comment:
Looks like, ITERATOR_PAGE_SIZE never used in the code. Please check and remove all code connected to this option if needed.